x-pack/plugins/ml/server/models/data_recognizer/modules/logs_ui_categories/ml/datafeed_log_entry_categories_count.json

```diff
@@ -1,15 +1,4 @@
 {
   "job_id": "JOB_ID",
-  "indices": ["INDEX_PATTERN_NAME"],
-  "query": {
-    "bool": {
-      "filter": [
-        {
-          "exists": {
-            "field": "message"
-          }
-        }
-      ]
-    }
-  }
```

Comment:
Since the setup API doesn't merge this and any queries supplied at setup time, the query is now only defined via the API.